storage/tianmu/mm/huge_heap_policy.cpp

```diff
@@ -62,7 +62,7 @@ HugeHeap::HugeHeap(std::string hugedir, size_t size) : TCMHeap(0) {
     rc_control_ << system::lock << "Huge Heap size (MB) " << (int)(size) << system::unlock;
     // size_ = size;
     // manage the region as a normal 4k pagesize heap
-    m_heap.RegisterArea(heap_frame_, size_ >> kPageShift);
+    m_heap_.RegisterArea(heap_frame_, size_ >> kPageShift);
```

Comment:
`m_heap_` --> `heap_`, the prefix `m_` is another naming style, here we follow google C++ style.